src/utilities/theme/types.ts

```diff
@@ -36,6 +36,10 @@ export interface RoleColors {
 
 export type Role = keyof RoleColors;
 
+export type ColorScheme = 'light' | 'dark';
+
+export type ThemeProviderColorScheme = ColorScheme | 'inverse' | undefined;
```

Comment:
Do we want something that should act as inherit from the current theme context? Is that what undefined does?

Reply:
`undefined` currently acts as `inherit`. Should we be more explicit and add that as a type?